concurrency-limits-core/src/main/java/com/netflix/concurrency/limits/limit/WindowedLimit.java

```diff
@@ -138,7 +141,8 @@ public void onSample(long startTime, long rtt, int inflight, boolean didDrop) {
         if (didDrop) {
             sample.updateAndGet(current -> current.addDroppedSample(inflight));
         } else {
-            sample.updateAndGet(current -> current.addSample(rtt, inflight));
+            long seqId = sequence.incrementAndGet();
```

Comment:
seqId is an implementation concerns of the percentile window and should not be part of a public API.